react.js

```diff
@@ -1,5 +1,6 @@
 const withReact = require('./lib/withReact')
 
 const babel = require('./babel')
+const withPrettier = require('./lib/withPrettier')
 
-module.exports = withReact(babel)
+module.exports = withPrettier(withReact(babel))
```

Comment:
Should use `base` to avoid applying `withPrettier` twice:

```js
const base = require('./base')

const withBabel = require('./lib/withBabel')
const withReact = require('./lib/withReact')
const withPrettier = require('./lib/withPrettier')

module.exports = withPrettier(withReact(withBabel(base)))
```